docs/docsite/rst/galaxy/user_guide.rst

```diff
@@ -364,9 +364,30 @@ Dependencies
 ------------
 
 Roles can also be dependent on other roles, and when you install a role that has dependencies, those dependencies will automatically be installed.
+When dependencies are encountered by ``ansible-galaxy``, it will automatically install each dependency to the ``roles_path``.
```

Comment:
This seems mostly redundant with the sentence above. Can you just add "to the ``roles_path``" to the end of the sentence above instead?